apps/dashboard/src/components/workflow-editor/steps/configure-step-form.tsx

```diff
@@ -78,6 +75,7 @@ export const ConfigureStepForm = (props: ConfigureStepFormProps) => {
     isReadOnly: isCodeCreatedWorkflow,
     save: (data) => {
       update(updateStepInWorkflow(workflow, data));
+      updateStepCache(data);
```

Comment:
because we use two separate endpoints: to update workflow and to fetch the step data, we need to update the step that is in the query cache on the `useFetchStep` hook